.travis.yml

```diff
@@ -13,7 +13,8 @@ notifications:
 cache: pip
 # install dependencies
 install:
-  - travis_wait pip install mindsdb
+  - pip install -r requirements.txt --quiet
```

Comment:
Thanks @wangshub . I think We should keep this and install mindsdb from pip. Also, travis_wait is important so Travis will not throw timeouts when installing large requirements as torch, numpy etc

Reply:
1. we should keep pip install from local to test newest version
2. travis_wait is added
@ZoranPandovski